db/re/item_db_usable.yml

```
@@ -78168,6 +78170,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_2,900000,10;
```

According to https://www.divine-pride.net/database/item/102781
```suggestion
      sc_start SC_CONTENTS_2,900000,10;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_SPELLBREAKER;
```